docs/contributing.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to the monorepo
+
+### Getting started
```

Comment:
Nit: This (and some other headings in this file) should be `##`.

```suggestion
## Getting started
```

Reply:
Whoops! Thank you.